.ci/test-install-docs.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+# Detect if any installation guides changed. If so, run the
+# kata manager to "execute" the install guide to ensure the
+# commands it specified result in a working system.
+check_install_guides()
+{
+	local -r install_guide_suffix="-installation-guide.md"
+
+	# List of filters used to restrict the types of file changes.
+	# See git-diff-tree(1) for further info.
+	local filters=""
+
+	# Added file
+	filters+="A"
+
+	# Copied file
+	filters+="C"
+
+	# Modified file
+	filters+="M"
+
+	# Renamed file
+	filters+="R"
+
+	# Unmerged (U) and Unknown (X) files. These particular filters
+	# shouldn't be necessary but just in case...
+	filters+="UX"
+
+	# List of changed files
+	local files=$(git diff-tree \
+		--name-only \
+		--no-commit-id \
+		--diff-filter="${filters}" \
+		-r \
+		origin/master HEAD || true)
+
+	# No files were changed
+	[ -z "$files" ] && return
+
+	changed=$(echo "$files" | grep "install/.*${install_guide_suffix}$" || true)
+
+	# None of the install guides were modified
+	[ -z "$changed" ] && return
+
+	echo "INFO: Found modified install guides: $changed"
+
+	# Regardless of which distro install guide(s) were changed, we test
+	# them all where possible.
+
+	local -r GOPATH=$(go env GOPATH)
+	[ -z "$GOPATH" ] && die "cannot determine GOPATH"
+
+	local -r mgr="${GOPATH}/src/github.com/kata-containers/tests/cmd/kata-manager/kata-manager.sh"
+
+	[ ! -e "$GOPATH" ] && die "cannot find $mgr"
+
+	source /etc/os-release
+
+	echo "INFO: Installing system from the $ID install guide"
+
+	$mgr install-packages
+
+	# Travis doesn't support VT-x
+	[ -n "$TRAVIS" ] && return
+
+	$mgr configure-image
+	$mgr enable-debug
+
+	msg="INFO: Successfully tested install guide for distro '$ID' $VERSION"
+
+	# Perform a basic test
+	sudo docker run --rm -i --runtime "kata-runtime" busybox echo "$msg"
+}
+
+main()
```

Comment:
the whole main calls main calls the only function in the file feels slightly redundant :-)

Reply:
Fixed.